tests/test_bootstrap.py

```diff
@@ -456,9 +456,9 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     assert order == [
         "http",
+        "an_after_dep",
```

Comment:
```suggestion
        "an_after_dep",  # After dependency of frontend and normal_integration
```

Reply:
On second thought, I think it's clear from the test why the order is the way it is